config/always-pyzfs.m4

```diff
@@ -18,18 +18,23 @@ AC_DEFUN([ZFS_AC_CONFIG_ALWAYS_PYZFS], [
 			DEFINE_PYZFS='--without pyzfs'
 		])
 	], [
-		DEFINE_PYZFS=''
+		AS_IF([USING_PYTHON], [
```

Comment:
This results in the following warning during `./configure --without-python`.

```
checking whether python2 version is >= 2.6... yes
checking for python2 version... 2.7
checking for python2 platform... linux2
checking for python2 script directory... ${prefix}/lib/python2.7/dist-packages
checking for python2 extension module directory... ${exec_prefix}/lib/python2.7/dist-packages
./configure: line 15946: USING_PYTHON: command not found
checking whether to enable pyzfs: ... no
```

Reply:
Whoops, thanks I missed that.